src/Data/Container/Morphism.agda

```diff
@@ -8,8 +8,8 @@
 
 module Data.Container.Morphism where
 
-open import Data.Container.Core
-import Function.Base as F
+open import Data.Container.Core using (Container; ⟦_⟧; _⇒_; shape; position)
+import Function.Base as F using (id; _∘′_)
```

Comment:
I realise that this is something else that I do not understand: why are we naming definitions that nevertheless need to be mentioned within qualified scope, as eg `F.id`? What advantage is there to explicitly listing them? (maybe some redundant recoding/documentary commentary advantage?)

Reply:
That's why *I* will sometimes do it. (There are also advantages when making copies of modules, but that's not the case here).  I had not asked for this particular use of explicit `using`, I think that's @jmougeot being 'thorough' while not noticing that these mean something slightly different than the usual cases.